README.md

```diff
@@ -115,6 +115,51 @@ For each in, shorthanded because manually writing hasOwnProperty each and every
 | obj | 
 | callback | 
 
+## checkFlags
+    A module to encapsulate bitmasking flags
```

Comment:
Document the limitations of bitmask flags. On some browsers, this limitation could be 32bits. On others, it could be 54. To see what the limitation is on those specific browsers, tests may need to be added and sugar methods may actually be useful here to detect bad scenarios. 

parseInt((Math.pow(2, 31)-1).toString(2),2) // 32bit support
parseInt((Math.pow(2, 53)-1).toString(2),2) // 54bit support

In both cases, on Chrome, the toString comes out as all 1s (on bits) and those can be converted back to the original value. Not all of these cases would be true on all browsers.